cogs/listeners.py

```diff
@@ -236,17 +236,20 @@ async def invite_in_message(self, message: discord.Message):
 
     @commands.Cog.listener('on_message')
     async def repo_mention(self, message: discord.Message):
-        if 'discord.py' in message.content and not message.author.bot:
-            regex = re.search(r'Rapptz/discord.py(#\d+)?', message.content)
-            if regex:
-                base_link = "https://github.com/Rapptz/discord.py"
-                group = regex.group(0)
-                if '#' in group:
-                    group = group.split('#')[1]
-                    base_link += f"/pull/{group}"
-                resp = await self.bot.session.get(base_link)
-                if resp.ok:
-                    await message.channel.send(base_link)
+        """
+        Format: repo: user/repo
+
+        Responds with a link to the repo.
+        """
+        if message.content.lower().startswith('repo:') and not message.author.bot:
+            repo = message.content.split('repo:')[1].strip()
+            if '/' not in repo:
+                return
+            user, repo = repo.split('/')
+            base_link = f"https://github.com/{user}/{repo}"
+            resp = await self.bot.session.get(base_link)
```

Comment:
Not really necessary. I'd prefer it removed. Maybe the repo is deleted or something, in that case i still want it to send link. (but mostly because of overhead). @Iusedmy1stnamelol what's your opinion?

Reply:
I think it can also be removed, I don't really see people finding a use.